### PR TITLE
quantumserverless rename

### LINKED ISSUE #780: Rename quantumserverless to quantum-serverless
### What is the expected behavior?

Due to not standardize our naming conventions we are generating errors like: #779 . I want to start introducing some conventions into this and use it along the code. My proposal is:
- quantum-serverless
- QuantumServerless

For those places where we require to reference the project. And this development should pontentially fix #779 .

### PR DESCRIPTION
**Pull request by Tansito**
### Summary

Fix #780 , #779

This PR pretends to standardize the use of `quantum-serverless` instead of `quantumserverless`.

### CODE REVIEW

**Comment by akihikokuroda** [comment]
I think that some changes need in `infrastructure/helm/quantum-serverless/templates/keycloakrealm.yaml` file.

File: infrastructure/helm/quantum-serverless/templates/keycloakrealm.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ data:
   keycloakconfig.json: |
     {
       "id": "e165160a-1516-4a29-b65c-6f4479682dc3",
-      "realm": "quantumserverless",
```
[the remark this answers is not in the document]
Comment (akihikokuroda) [reply infrastructure/helm/quantum-serverless/templates/keycloakrealm.yaml:7]: Oh, I did see these changes.

**Comment by psschwei** [comment]
This failed when running on kubernetes:

`INFO 2023-07-17 14:15:52,403 update_jobs_statuses.py:57 : Job [cae382c9-d1ba-4b7f-b87a-5a0e1abbcafa] status changed from [PENDING] to [FAILED]`

(of course the ray pods shut down before I checked their logs, to be continued...)

**Comment by psschwei** [comment]
nothing in the ray pods, keycloak, or scheduler to indicate why it failed... :shrug:

**Comment by akihikokuroda** [comment]
@psschwei @Tansito I fetched this branch and rebuilt all images.  After that It is working fine for me.

**Comment by Tansito** [comment]
I tested it too with one of the programs and for me it's working.

**Review by akihikokuroda — APPROVED** [review]
LGTM

**Review by pacomf — APPROVED** [review]

**Pull request merged**